tools/deployment/package/docker-compose.runtime.yaml

```diff
@@ -0,0 +1,34 @@
+services:
+  clp-runtime:
+    hostname: "clp_runtime"
+    image: "${CLP_PACKAGE_CONTAINER_IMAGE_REF:-clp-package}"
+    logging:
+      driver: "local"
+    stdin_open: true
+    tty: true
+    user: "${CLP_FIRST_PARTY_SERVICE_UID_GID:-1000:999}"
+    environment:
+      HOME: "${HOME:?Please set a value.}"
```

Comment:
```suggestion
      # NOTE: We forward "$HOME" into the container so that if the user specified "~" anywhere in
      # their config, the container can resolve it to the relevant path on the host. As a result,
      # the container should not rely on "$HOME" or "~" pointing to a path inside the container.
      HOME: "${HOME:?Please set a value.}"

```